### ISSUE TITLE
Convert private class members from `private member` to `#member`

### ISSUE THREAD

**Issue by d13** [gitkraken/vscode-gitlens#3872]
The `private` keyword is fine for TypeScript during development, but doesn't enforce `private` access at runtime where prefixing a class member with `#` will add the runtime enforcement.

See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties

**Comment by eamodio** [comment]
I don't think we should do this, as there is a perf penalty for private members, unless there are specific cases that we NEED to keep private (certainly anything we expose outside ourselves).
